common.props

```diff
@@ -7,7 +7,7 @@
     <Copyright>Copyright © 2004-2020 OPC Foundation, Inc</Copyright>
     <Company>OPC Foundation</Company>
     <Authors>OPC Foundation</Authors>
-    <NoWarn>NU5125</NoWarn>
+    <NoWarn>NU5125;CS1591;CS1587;RCS1138</NoWarn>
```

Comment:
You should remove this warning, because GenerateDocumentationFile isn't available.

Reply:
you spot everything :-) I'm working on the XML comments, not supposed to be in this PR yet..

Reply:
😎